Source/Engine/UI/GUI/ContainerControl.cs

```diff
@@ -98,12 +97,19 @@ public bool IsLayoutLocked
         /// <summary>
         /// Gets or sets a value indicating whether apply clipping mask on children during rendering.
         /// </summary>
-        [EditorOrder(530), Tooltip("If checked, control will apply clipping mask on children during rendering.")]
-        public bool ClipChildren
-        {
-            get => _clipChildren;
-            set => _clipChildren = value;
-        }
+        [HideInEditor, Tooltip("If checked, control will apply clipping mask on children during rendering."), Obsolete("Use Clipping"), NoSerialize]
+        public bool ClipChildren;
+
+        /// <summary>
+        /// Controls how the clipping behavior of this Control.
+        /// Normally content that overflows the bounds of the Control continues rendering.
+        /// Enabling clipping prevents that overflowing content from being seen.
+        /// <para><b>Note:</b></para>
+        /// Controls in different clipping spaces can not be batched together, and so there is a performance cost to clipping.
+        /// Do not enable clipping unless a panel actually needs to prevent content from showing up outside its bounds.
+        /// </summary>
+        [EditorOrder(530), Tooltip("Controls how the clipping behavior of this Control.\nNormally content that overflows the bounds of the Control continues rendering.\nEnabling clipping prevents that overflowing content from being seen.\n\nNote:\nControls in different clipping spaces can not be batched together, and so there is a performance cost to clipping.\nDo not enable clipping unless a panel actually needs to prevent content from showing up outside its bounds.")]
+        public Clipping Clipping;
```

Comment:
Initialize it to the `Clipping.Inherit` so it's more explicit and add DefaultValue(Clipping.Inherit)` attribute so will have hint on modification in Editor.

Reply:
ok

Reply:
@mafiesto4  done